test/integration/targets/command_shell/tasks/main.yml

```diff
@@ -167,6 +179,32 @@
     that:
       - command_result3 is not changed
 
+# changes
+
+- name: Verify that afile.txt content has changed
+  assert:
+    that: "'win' in lookup('file', '{{ output_dir_test }}/afile.txt') | b64encode | b64decode"
+
+- name: Changes afile.txt content
+  command: "{{ output_dir_test }}/change_afile.sh {{output_dir_test }}/afile.txt"
+  args:
+    changes: "{{ output_dir_test }}/afile.txt"
+
+- name: Verify that afile.txt content has changed
+  assert:
+    that: "'won' in lookup('file', '{{ output_dir_test }}/afile.txt') | b64encode | b64decode"
```

Comment:
A minor nit: group all the `assert` cases at the end of the test rather than interspersing them.